cmd/connector/main.go

```diff
@@ -55,36 +54,33 @@ func main() {
 	defer client.Close()
 
 	// Watcher
-	watch := kv.NewWatcher(
-		namespace.NewKV(client, connectionsPrefix),
-		namespace.NewWatcher(client, connectionsPrefix),
-		namespace.NewKV(client, locksPrefix),
-		logger.Named("KV.Watcher"))
+	watch := kv.NewWatcher(client, logger.Named("KV.Watcher"))
 	events, err := watch.Watch()
 	if err != nil {
 		logger.Fatalf("unable to watch changes in etcd. Error: %s", err)
 	}
 	defer watch.Stop()
 
+	// KV store service
+	store := kv.NewStore(namespace.NewKV(client, kv.GetPrefix()), jobsBucketSize, logger.Named("KV.Store"))
```

Comment:
I think it will be better to just pass bare client there and namespace it inside NewStore.